Core/src/com/serotonin/m2m2/rt/event/handlers/EmailHandlerRT.java

```diff
@@ -205,6 +205,13 @@ synchronized public void scheduleTimeout(EventInstance evt, long fireTime) {
         }
     }
 
+    @Override
+    synchronized public void terminate(EventInstance evt) {
```

Comment:
I just noticed this but escalationTask is set in a non-synchronized method.  So there is potential for an event to be made active and schedule the escalation all while terminate() is called and doesn't see the task set yet....  This would leave scheduled tasks in the system.

Reply:
@terrypacker I reviewed the code with Mert and I'm sure that would not happen, can you please elaborate on your concern? I'm approving this for testing.